src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -87,7 +120,74 @@ def ramp_fit(model, buffsize, save_opt, readnoise_2d, gain_2d,
         Object containing optional GLS-specific ramp fitting data for the
         exposure
     """
+    # Create an instance of the internal ramp class, using only values needed
+    # for ramp fitting from the to remove further ramp fitting dependence on
+    # data models.
+    ramp_data = create_ramp_fit_class(model, dqflags)
+
+    return ramp_fit_data(
+        ramp_data, buffsize, save_opt, readnoise_2d, gain_2d,
+        algorithm, weighting, max_cores, dqflags)
+
+
+def ramp_fit_data(ramp_data, buffsize, save_opt, readnoise_2d, gain_2d,
+                  algorithm, weighting, max_cores, dqflags):
+    """
+    This function begins the ramp fit computation after the creation of the
+    RampData class.  It determines the proper path for computation to take
+    depending on the choice of ramp fitting algorithms (which is only ordinary
+    least squares right now) and the choice of single or muliprocessing.
+
+
+    ramp_data: RampData
+        Input data necessary for computing ramp fitting.
+
+    buffsize : int
+        size of data section (buffer) in bytes
+
+    save_opt : bool
+       calculate optional fitting results
+
+    readnoise_2d: ndarray
```

Comment:
Here ans else where leave a space bwtween the parameter name and `:`.

Reply:
I updated the docstrings in this file.